FRAMEWORK-OVERVIEW.md

```diff
@@ -82,7 +83,12 @@ The event delegates are the ones that add behaviour to certain events inside a s
 * [SignupControllerTransitionDelegate](Authentication/SignUp/View/SignupControllerTransitionDelegate.swift)
 
 
+## Using Custom Views
 
+If the user wants to use custom views for Login, Sign Up or both, he or she should create a class that implements `AuthenticationComponentsFactoryType` and override the correspondent method (`createLoginView(withDelegate delegate: LoginViewDelegate, loginProviders: [LoginProvider]) -> LoginViewType` for example) and pass it to the `AuthenticationCoordinator` on creation.
```

Comment:
`If you want to use ...`
I'd use second person everywhere.

Reply:
I thought of that but the rest of the overview is written using "the user", should I change it everywhere?

Reply:
:O you are right ! If you want, you are more than welcome. If not, leave it and add an issue.